app/tree/gui/src/ts/twxplore/gui/geo/components/TreeCollapse/TreeCollapse.tsx

```diff
@@ -0,0 +1,112 @@
+import './TreeCollapse.scss';
+import * as React from "react";
+import { Theme, createStyles, makeStyles } from '@material-ui/core/styles';
+import {ListGroup, ListGroupItem} from "reactstrap";
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+//import * as PropTypes from "prop-types";
+
+
+
+type Feature_List = {
+  createdAt: string,
+  dbh: number,
+  stump: number,
+  curbLoc: string,
+  status: string,
+  health: string | null,
+  species: string | null,
+  steward: string | null,
+  guards: string | null,
+  sidewalk: string | null,
+  userType: string,
+  problems: string[],
+  address: string,
+  city: string,
+  community: number,
+  cncldist: number,
+  stateAssembly: number,
+  stateSenate: number,
+  boroughCount: number,
+  latitude: number,
+  longitude: number,
+  x_sp: number,
+  y_sp: number,
+  bin: number | null,
+  bbl: any | null,
+  uri: string
+}
+
+type TreeCardProps = {
+  features: Feature_List,
+  callSetMode: Function
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      width: '100%',
+    },
+    heading: {
+      fontSize: theme.typography.pxToRem(15),
+      fontWeight: theme.typography.fontWeightRegular,
+    },
+  }),
+);
+
+
+
+
+export const TreeCollapse: React.FunctionComponent<TreeCardProps> = ({features, callSetMode}) => {
+    const classes = useStyles();
+
+
+    const handleClick = (e) => {
+      e.preventDefault()
+      if (e.target.id == 'species'){
```

Comment:
You shouldn't normally need to do this target checking. You can create the onClick function within the map, which will capture key in its closure. Will notate below.

Reply:
Created onClick function in map.